generator/golang/scope_internal.go

```diff
@@ -30,7 +30,9 @@ const (
 	// A prefix to denote synthesized identifiers.
 	prefix = "$"
 	// nestedAnnotation is to denote the field is nested type.
-	nestedAnnotation = "thrift.nested"
+	nestedAnnotation    = "thrift.nested"
+	interfaceAnnotation = "thrift.interface"
```

Comment:
I suggest `is_interface` & `is_alias`, which is easiler to understand

Reply:
done.